DAO.sol

```diff
@@ -281,11 +281,22 @@ contract DAO is DAOInterface, Token, TokenSale {
 
     function vote(uint _proposalID, bool _supportsProposal) onlyTokenholders noEther returns (uint _voteID) {
         Proposal p = proposals[_proposalID];
-        if (p.voted[msg.sender] || now >= p.votingDeadline) throw;
```

Comment:
Because of https://github.com/slockit/DAO/pull/17/files#diff-b30f9d3be2a832949bdd72e23956c130R224 there can never be a proposal with 'ID = 0' . Therefore the ' votingDeadline = 0' and it throws.